code/about.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Project Name</title>
+    <link rel="stylesheet" type="text/css" href="style.css"/>
+  </head>
+  <body>
+    <header>
+      <menu>
+        <a href="./index.html">Home</a>
+        <a href="./photos.html">Photo Gallery</a>
```

Comment:
I like that you have included a navbar! 

I would recommend to use the full navbar for each of the HTML-pages, in this case I would also recommend to include About in the navbar and for example to style it in CSS if the user is on that page, for example:
`<a class="active-nav" ...>` and in CSS use a different color and font-weight for .active-nav.